src/plot_api/helpers.js

```diff
@@ -484,3 +484,19 @@ exports.manageArrayContainers = function(np, newVal, undoit) {
         np.set(newVal);
     }
 };
+
+var ATTR_TAIL_RE = /(\.[^\[\]\.]+|\[[^\[\]\.]+\])$/;
```

Comment:
(not a big deal; just difficult to confirm we're on the same page about what the end of an attr string means by mentally parsing the regexp)